voting-scheme-dummy/src/voter_wrapper_adapter.js

```diff
@@ -0,0 +1,43 @@
+import { VoterWrapper } from "./voter_wrapper";
+
+/**
+ * This is just a dummy implementation of a possible `VoterWrapperAdapter`.
+ * It is based on the dummy voting scheme that we are using in the Bulletin Board.
+ */
+export class VoterWrapperAdapter {
+  /**
+   * Initializes the class with the given params.
+   *
+   * @constructor
+   * @param {Object} params - An object that contains the initialization params.
+   * - {String} voterId - The unique id of a trustee.
+   */
+  constructor({ voterId }) {
+    this.voterId = voterId;
+    this.wrapper = new VoterWrapper({ voterId });
+  }
+
+  /**
+   * Process the message and update the wrapper status.
```

Comment:
```suggestion
   * Processes the message and updates the wrapper status.
```

Reply:
Discussed on slack to apply them in #110